projects/storefrontapp-e2e-cypress/cypress/e2e/regression/asm/asm.emulation.core-e2e.cy.ts

```diff
@@ -78,11 +78,41 @@ context('Assisted Service Module', () => {
       );
     });
 
-    // TODO(#9445): Add e2e test for this scenario
-    it.skip('agent login when user is logged in should start this user emulation', () => {});
+    it.skip('agent login when user is logged in should start this user emulation', () => {
```

Comment:
This run result is not what we expected, after the agent logs in, customer seesion will sign out and not automatically start this custormer emulation, so I can't enable this case currently.